packages/shared/providers.yaml

```diff
@@ -407,6 +407,28 @@ brex-staging:
     proxy:
         base_url: https://platform.staging.brexapis.com
     docs: https://docs.nango.dev/integrations/all/brex
+builder-io-private:
+    categories:
+        - dev-tools
+        - design
+        - cms
+    auth_mode: API_KEY
+    proxy:
+        base_url: https://${connectionConfig.domain}
+        headers:
+            Authorization: Bearer ${apiKey}
+    docs: https://docs.nango.dev/integrations/all/builder-io
+builder-io-public:
+    categories:
+        - dev-tools
+        - design
+        - cms
+    auth_mode: API_KEY
+    proxy:
+        base_url: https://${connectionConfig.domain}
+        query:
+            apiKey: ${apiKey}
```

Comment:
does they really have 2 API with 2 ways to pass the api key?

Reply:
Yes, they provide users with both `private` and `public` keys. For `public` keys, you append them to the query parameters, while `private` keys are added to the authorization header, as follows; 
![image](https://github.com/user-attachments/assets/7b6ed1a8-d663-4754-8099-36e0ae2fecfa)
![image](https://github.com/user-attachments/assets/1bcc78ac-b1f6-4a7e-9d13-90ef61358509)

Reply:
Yes, using the public API key appended to the URL query parameters should help avoid CORS issues on the frontend.